cmd/testrunner.go

```diff
@@ -258,6 +289,10 @@ func testTypeCommandActionFactory(runner testrunner.TestRunner) cobraext.Command
 				ServiceVariant:     variantFlag,
 				WithCoverage:       testCoverage,
 				CoverageType:       testCoverageFormat,
+				ConfigFilePath:     configFileFlag,
+				RunSetup:           runSetup,
```

Comment:
`elastic-package service up` fails now after a setup is done, because the container already exists. Should we do something about this?

I see these options:
* Make `elastic-package service up` aware of the service started by `elastic-package test --setup`. And attach to it if started?
* Remove `elastic-package service up`. Its purpose was probably the same as `elastic-package test --setup` has now, but `elastic-package test --setup` is more complete as it also configures the agent, and has a way to tear down. Though it is true that their use cases might be slightly different.
* Keep both commands, but make them explicitly fail if the other one has been used to start a service.

In any case we don't need to solve this now.

Reply:
Current error is:

```
 $ elastic-package service up 
Boot up the service stack
2024/02/14 16:57:44  INFO Using service variant: ServiceVariant{Name: v1, Env: SERVICE_VERSION=1.19.5}
Error: up command failed: can't set up the service deployer: can't attach service container to the stack network: could not attach container to the stack network (stderr="Error response from daemon: endpoint with name elastic-package-service-nginx-1 already exists in network elastic-package-stack_default\n"): exit status 1
```

It does not give us too much information of the root cause of the issue (the service is already running), but as you mention I would leave it for a future change.